src/Sylius/Component/Payment/PaymentTransitions.php

```diff
@@ -22,6 +22,7 @@ final class PaymentTransitions
     public const TRANSITION_COMPLETE = 'complete';
     public const TRANSITION_FAIL = 'fail';
     public const TRANSITION_CANCEL = 'cancel';
+    public const TRANSITION_AUTHORIZE = 'authorize';
```

Comment:
We should update also [this doc](https://docs.sylius.com/en/latest/book/orders/payments.html#payment-state-machine)